source/agora/consensus/Validation.d

```diff
@@ -990,6 +993,62 @@ public string isInvalidReason (const ref Block block, in ulong prev_height,
     return null;
 }
 
+/*******************************************************************************
+
+    Check the validator of an enrollment.
+
+    A Validator's enrollment is considered valid if:
+        - UTXO is unspent frozen utxo
+        - Signatures are authentic
+        - The frozen amount must be equal to or greater than 40,000 BOA
+        - The block height shall be greater than the unlock_height of the UTXO.
```

Comment:
`shall be greater than the unlock_height` => `must be at least unlock_height or greater`

Reply:
applied the suggest.